pylua/test/test_timer.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+
+import timeit
+import threading
+import pylua.timer as timer
+
+
+class TestPyTimer:
+
+    def _timer_task(self):
+        self._task_is_run = True
+        self._event.set()
+
+    def __init__(self):
+        self._interval = 50  # interval in milliseconds
+        self._timer = timer.PyTimer(self._timer_task, self._interval)
+        self._task_is_run = False
+        self._event = threading.Event()
+
+    def _wait_event(self, wait_time):
+        wait_time = timer.msec_to_sec(wait_time)
+        self._event.clear()
```

Comment:
@ dtrunov, yes, python impl of event has a timeout and if for some reason there will problem I've described, then after timeout wait will be unblocked. But it's  icorrect to rely o. Timeout in this case. There is common pattern, which apllicable to any impl. Stick with it. Usualy timeout is qute big and this some fallback logic to prevent a deadlock. SInce your  test doesnt chech edge case, please set timeoout here to some infinite value. I want with method to forever if something will go wrong. TRavis has1 h timeout as far as i know so after some time build will fail in the worst case.